taxi/urls.py

```diff
@@ -16,11 +23,55 @@
         ManufacturerListView.as_view(),
         name="manufacturer-list",
     ),
-    path("cars/", CarListView.as_view(), name="car-list"),
-    path("cars/<int:pk>/", CarDetailView.as_view(), name="car-detail"),
-    path("drivers/", DriverListView.as_view(), name="driver-list"),
     path(
-        "drivers/<int:pk>/", DriverDetailView.as_view(), name="driver-detail"
+        "manufacturers/create",
```

Comment:
```suggestion
        "manufacturers/create/",
```
add trailing `/`
also please do it in other urls